### PR TITLE
Fixed size array deserialization

### LINKED ISSUE #74: bincode deserialization with serde might be broken
Example of some code that fails to run:
https://gist.github.com/iqualfragile/743cc39ac4199e0f669bae13b778474e

### PR DESCRIPTION
**Pull request by 0nkery**
Fixes #74. 

[This](https://github.com/serde-rs/serde/pull/392) PR in `serde` fixes this problem but I don't know when and how they will release patch.

### CODE REVIEW

File: tests/test.rs [hunk 1]
```diff
@@ -216,6 +216,14 @@ fn test_unicode() {
 }
 
 #[test]
+fn test_fixed_size_array() {
+    the_same([24u32; 32]);
+    the_same([1u64, 2, 3, 4, 5, 6, 7, 8]);
+    the_same([0u8; 19]);
+}
+
+
```
Comment (dtolnay) [inline tests/test.rs]: Remove the extra newline

**Comment by dtolnay** [comment]
This LGTM. We will fix this upstream in Serde in the next breaking release.

**Comment by 0nkery** [comment]
@TyOverby Tests aren't building due to #77.

**Comment by TyOverby** [comment]
Checks look good!  Thanks!

**Pull request merged**